tools/view_model.cpp

```diff
@@ -284,27 +295,30 @@ int main(int argc, char **argv)
     }
 
     double dist = 2 * std::max(p_max.z - p_min.z, std::max(p_max.x - p_min.x, p_max.y - p_min.y));
-    double h = (p_max.z - p_min.z) / 2;
 
-    std::cout << "Model loaded successfully:" << std::endl;
-    std::cout << "    " << n_vertices << " vertices" << std::endl;
-    std::cout << "    " << n_triangles << " triangles" << std::endl;
+    std::stringstream info_msg;
+    info_msg << "Model loaded successfully:" << std::endl;
+    info_msg << "    " << n_vertices << " vertices" << std::endl;
+    info_msg << "    " << n_triangles << " triangles" << std::endl;
+
+    info_msg << std::endl;
+    info_msg << "Mouse:" << std::endl;
+    info_msg << "    left         - orbit" << std::endl;
+    info_msg << "    middle       - zoom" << std::endl;
+    info_msg << "    right        - pan" << std::endl;
+    info_msg << "    double click - fly to" << std::endl;
 
-    std::cout << std::endl;
-    std::cout << "Mouse:" << std::endl;
-    std::cout << "    left         - orbit" << std::endl;
-    std::cout << "    middle       - zoom" << std::endl;
-    std::cout << "    right        - pan" << std::endl;
-    std::cout << "    double click - fly to" << std::endl;
+    info_msg << std::endl;
+    info_msg << "Keys:" << std::endl;
+    info_msg << "    r - reload model" << std::endl;
+    info_msg << "    v - show / hide model volumes" << std::endl;
+    info_msg << "    c - circle rotate" << std::endl;
+    info_msg << "    p - snap pitch" << std::endl;
+    info_msg << "    q - quit" << std::endl;
 
-    std::cout << std::endl;
-    std::cout << "Keys:" << std::endl;
-    std::cout << "    r - reload model" << std::endl;
-    std::cout << "    a - show / hide model areas" << std::endl;
-    std::cout << "    c - circle rotate" << std::endl;
-    std::cout << "    q - quit" << std::endl;
+    std::cout << info_msg.str();
```

Comment:
Why not dump a multi-line C string in to cout?

```` cpp
cout << "blah\n"
            "more blah\n"
            "etc\n";
````
Since 2 strings after each other simply get concatenated.

Reply:
I don't know. This is already better then the previous implementation.